it/common/src/main/java/com/google/cloud/teleport/it/common/utils/ResourceManagerUtils.java

```diff
@@ -123,8 +123,8 @@ public static void checkValidProjectId(String idToCheck) {
       throw new IllegalArgumentException(
           "Project ID "
               + idToCheck
-              + " is not a valid ID. Only letters, numbers, hyphens, single quotes and exclamation"
-              + " points are allowed.");
+              + " is not a valid ID. Only letters, numbers, hyphens, single quotes, colon and"
+              + " exclamation points are allowed.");
```

Comment:
I think you're right. I've been pretty confused and mechanically added this change, because there was some issue with colons in the past, but I think it was fixed at some point

Reply:
This ILLEGAL_PROJECT_CHARS is a negation of legal chars, therefore colon should be there. Our legacy project is named google.com:cloud-bigtable-dev